### PR TITLE
Clarify that 2 PATs are required for sourcemaps

### PR DESCRIPTION
**Pull request by captainbrosset**
When publishing/obtaining source maps to/from the Azure Artifacts symbol server, 2 personal access tokens are required. And they are not the same. One will likely be created by the CI admin when setting up the Azure pipeline that generates the source maps. And the other one will be created by each devtools users who want to retrieve the published source maps from devtools.

This PR makes a couple of minor changes to the doc to make this a bit more clear.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ To learn about how to publish source maps, see [Securely debug original code by
 
 To obtain source maps from the Azure Artifacts symbol server, DevTools uses the fetch protocol to talk to Azure DevOps, which requires a valid Personal Access Token (PAT).
 
+You need to generate a PAT even if you already generated one to publish source maps to the symbol server in [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md).
+
 To generate a PAT in Azure DevOps:
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:36]: Should the image after step 2 and the text in step 3 match the changes to from the Publish version in the other changed file? The text and image don't exactly match here either ... image is "devtool source maps" vs step 3 "DevTools source maps" so we can fix this wording or replace both with the new version.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:36]: @captainbrosset @vchapel 

discussion from other PR: https://github.com/MicrosoftDocs/edge-developer/pull/1766#discussion_r818468903
I imagine giving some specific advised, suggested names such as this breakout:

Publish page's image: "DevTools source maps - Admin PAT" or "Admin PAT for DevTools source maps"
Publish page's text: "DevTools source maps - Admin PAT" or "Admin PAT for DevTools source maps"

Consume page's image: "DevTools source maps - User PAT" or "User PAT for DevTools source maps"
Consume page's text: "DevTools source maps - User PAT" or "User PAT for DevTools source maps"

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:36]: @vchapel you're right, the text on the Consume page didn't exactly match the image. The text was "DevTools source maps" while the image showed "devtool source maps". I fixed that.

> Should the image after step 2 and the text in step 3 match the changes to from the Publish version in the other changed file?

I'm not following you here. The PAT created in the Publish page has nothing to do with the PAT on the Consume page. Those are 2 different things.

@mikehoffms the 2 pages have the advices now:

* Publish page has: `enter a name for the PAT, such as "publish source maps"`
* Consume page has: `enter a name for the PAT, such as "devtool source maps"`

And both images show these strings too.

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**